src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dns/ClusterDnsCache.scala

```diff
@@ -65,6 +67,10 @@ class ClusterDnsCache(proxyConfig: ProxyConfig, dbRef: DbReference) extends Acto
       processClusters(clusters)
       scheduleRefresh
 
+    // add one ready cluster to the caches now instead of waiting for the refresh cycle
```

Comment:
Does not interrupt or reset the refresh cycle, and this cluster's cache entries will be repopulated at the next iteration of the cycle